core/domain/exp_jobs_one_off.py

```diff
@@ -30,10 +31,13 @@
 import feconf
 import utils
 
-(base_models, exp_models,) = models.Registry.import_models([
-    models.NAMES.base_model, models.NAMES.exploration])
+(file_models, base_models, exp_models,) = models.Registry.import_models([
+    models.NAMES.file, models.NAMES.base_model, models.NAMES.exploration])
 
 _COMMIT_TYPE_REVERT = 'revert'
+# pylint: disable=line-too-long
+ALLOWED_IMAGE_EXTENSIONS = [extension for formats, extensions in feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS.iteritems() for extension in extensions]
```

Comment:
This is a bit unclear -- too many dummy variables floating around, and `formats` is unused. Consider using itertools.chain() on feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS.values().

Reply:
Done!